gradle-talend-component/src/main/java/org/talend/sdk/component/gradle/TaCoKitPlugin.java

```diff
@@ -149,5 +150,16 @@ public void apply(final Project project) {
                 put("description", "Creates a Component ARchive (.car) based on current project.");
             }
         }, "talendComponentKitComponentArchive");
+
+        // deploy in studio
+        project.task(new HashMap<String, Object>() {
+
+            {
+                put(Task.TASK_TYPE, DeployInStudioTask.class);
```

Comment:
On the style can you please use direct string as others or update others with these constants please?

Reply:
Sure thing, updated all the occurrences to use constants. 👍